.github/workflows/check-release.yml

```diff
@@ -9,10 +9,10 @@ jobs:
   check-version:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
 
     - name: Set up Python
-      uses: actions/setup-python@v3
+      uses: actions/setup-python@v5
       with:
         python-version: '3.10'
```

Comment:
For demo we use python 3.9 and here we use 3.10. May be we need to stick with one python version ?

Reply:
Done.